src/infrastructure/config/schema.js

```diff
@@ -0,0 +1,41 @@
+const SimpleSchema = require('simpl-schema').default;
+const { validateConfigAgainstSchema, schemas, patterns } = require('login.dfe.config.schema.common');
+const config = require('./index');
+const logger = require('./../logger');
+
+const notificationsSchema = new SimpleSchema({
+  connectionString: patterns.redis,
+});
+
+const adapterSchema = new SimpleSchema({
+  type: {
+    type: String,
+    allowedValues: ['file', 'redis', 'mongo', 'azuread', 'sequelize'],
+  },
+  directories: {
+    type: schemas.sequelizeConnection,
+    optional: true,
+  },
+  organisation: {
+    type: schemas.sequelizeConnection,
+    optional: true,
+  },
+});
+
+const schema = new SimpleSchema({
```

Comment:
can you make sure we need all those? Are all these properties part of the config for public API?